src/TransactionHistory/TransactionHistory.jsx

```diff
@@ -0,0 +1,114 @@
+import React, {useState, useEffect} from "react";
+import PropTypes from "prop-types";
+import Typography from "@mui/material/Typography";
+import Box from "@mui/material/Box";
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Tooltip,
+} from "@mui/material";
+import * as Icons from "@mui/icons-material";
+import IconButton from "@mui/material/IconButton";
+
+const TransactionHistory = (props) => {
+  const [copyText, setCopyText] = useState("Copy");
+  const [open, setOpen] = useState(false);
+  
+  const handleTooltipClose = () => {
+    setTimeout(() => {
+      setOpen(false);
+      setCopyText("Copy");
+    }, 200);
+  }
+
+  const handleTooltipOpen = () => {
+    setOpen(true);
+  }
+
+  return (
+    <React.Fragment>
+    {props.history.length === 0 && (
+      <Box sx={{ p: 3 }}>
+        <Typography component={'span'} variant="h6" align="center">
+          No history
+        </Typography>
+      </Box>
+    )}
+    {props.history.length > 0 &&
+      props.history.map((item, index) => {
+        const accordionTitle = `
+          [block:${item.blockNumber} txIndex:
+            ${parseInt(item.transactionIndex, 16)}] from: ${item.from.substring(
+          0,
+          5
+        )}...${item.from.substring(
+          item.from.length - 5,
+          item.from.length
+        )} to:${" "}
+            ${item.to}
+            logs: ${item.logs.length}`;
+
+        return (
+          <Accordion>
```

Comment:
accordion should have a "key={index}". React needs keys when returning things in a map